src/app/components/compareNI.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable react/jsx-fragments */
+/* eslint-disable react/jsx-filename-extension */
+import React, { useState } from 'react';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+import Container from 'react-bootstrap/Container';
+import Card from 'react-bootstrap/Card';
+import Row from 'react-bootstrap/Row';
+import Col from 'react-bootstrap/Col';
+import Alert from 'react-bootstrap/Alert';
+import axios from 'axios';
+import styles from './compareNI.module.css';
+
+export default function CompareNI() {
+  const [NI18, setNI18] = useState('Enter your salary');
+  const [NI19, setNI19] = useState('Enter your salary');
+  const [NIdiff, setNIdiff] = useState('Enter your salary');
+  const [loaded, setLoaded] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  async function getNI(income, date) {
+    const options = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        'x-run-date': date,
+      },
+    };
+    try {
+      const { data } = await axios.post('/v1/national-insurance', { income }, options);
+      return data.ni;
+    } catch (err) {
+      setIsError(true);
+      throw err;
+    }
+  }
+
+  async function handleSubmit(event) {
+    event.preventDefault();
+    const salary = event.target.elements.salary.value;
+
+    const ni1 = await getNI(salary, '2018-04-06');
+    const ni2 = await getNI(salary, '2019-04-06');
+    const diff = parseFloat(ni2 - ni1).toFixed(2);
+    setIsError(false);
+
+    setNI18('£'.concat(ni1));
+    setNI19('£'.concat(ni2));
+    setNIdiff(diff > 0 ? '+ £'.concat(diff) : '- £'.concat(diff.toString().substring(1)));
+    setLoaded(true);
+  }
+
+  return (
+    <>
+      <Container fluid className={styles.titleContainer}>
+        <h1 className={styles.title}>Compare Your National Insurance Contributions</h1>
+      </Container>
+      <Container className={styles.contentContainer}>
+        <Form onSubmit={handleSubmit}>
+          <Form.Group as={Row}>
+            <Form.Label column sm="1" className={styles.salaryInputLabel}>Salary</Form.Label>
+            <Col sm="11">
+              <Form.Control
+                size="lg"
+                type="text"
+                name="salary"
+                placeholder="Enter your salary"
+              />
```

Comment:
I liked that you reached for a `label` element to caption the `input` and put a placeholder on the input!

To make this even better and more semantically correct it'd be awesome to associate the two for screen readers -- right now if a screen reader was to use this (usually used by someone who's blind/with partial sight) then they wouldn't know the salary label and input are associated. You could do this with a `for` attribute (`htmlFor` in React) and an `id` on the `input`. There's some more details [here](https://www.w3.org/WAI/tutorials/forms/labels/) if you're interested 😄 
```suggestion
            <Form.Label htmlFor="salary-input" column sm="1" className={styles.salaryInputLabel}>Salary</Form.Label>
            <Col sm="11">
              <Form.Control
                id="salary-input"
                size="lg"
                type="text"
                name="salary"
                placeholder="Enter your salary"
              />
```